docker/install_rust.sh

```diff
@@ -2,17 +2,16 @@
 
 set -eux
 
-apt-get update
-apt-get install -y --no-install-recommends \
+apt install -y --no-install-recommends \
         ca-certificates \
         gcc \
         libc6-dev \
         wget \
         libssl-dev \
         pkg-config \
+        build-essential
```

Comment:
Added this (for now, at least), to bypass this:
```
(...)
   Compiling tokio v0.2.20
   Compiling futures-util v0.3.4
   Compiling thiserror v1.0.16
   Compiling tracing v0.1.13
   Compiling futures-executor v0.3.4
   Compiling futures v0.3.4
error: failed to run custom build command for `libsodium-sys v0.2.5`

Caused by:
  process didn't exit successfully: `/target/debug/build/libsodium-sys-2c273b8b56cf939d/build-script-build` (exit code: 101)
```

Reply:
seems legit.